doc/RELEASE_GUIDE.md

```diff
@@ -9,3 +9,79 @@ You can update Play Store releae metadata using `fastlane android upload_metadat
 But that requires credentials necessary for accessing Play Store API. The simpler way is to edit files contained within [`fastlane/metadata`](metadata) and run the following CI job:
 
 https://ci.status.im/job/status-tools/job/update-playstore-metadata/
+
+# Release process
+
+## Coordination
+
+All the coordination is done in the #release channel on discord.
+
+## Create a branch
+
+First we need to create a release branch cut from develop.
+The name of the branch should be `release/a.b.x` where `x` is actually `x`, not 
+a variable, and `a` and `b` are the `MAJOR` and `MINOR` version of the app.
+
+For example, valid branch names are:
+
+`release/1.12.x` or `release/3.12.x`
+
+
+## Update version
+
+We need to update the `VERSION` file to reflect the release we want to build.
```

Comment:
Shouldn't the `VERSION` in `develop` be the next version, not the last version?
Or if not the next version the the release candidate of the next version: `1.13.0-rc`
And then when you create the `release/1.13.x` branch you update the `VERSION` in it to be `1.13.0` and the tone in `develop` to be `1.14.0-rc`. Or something like that. Because we notoriously forget to update `VERSION` in `develop`.

Reply:
What if we just create a script that cut the release branch and update version?

`scripts/cut-release-branch.sh`

and it:
1) Creates a branch `release/x.y.z`
2) Bump version to `x.y.z`
3) Creates a PR in status-react with the updated version?

That should be easy (maybe 3 requires some manual intervention, not sure)

Reply:
This way we avoid a lot of manual mistakes that can happen and make sure all the three steps are always taken.

Reply:
Do you want to make it or should I?

Reply:
I can make it!

Reply:
Maybe to not confuse actual release candidates and nightlies/dev builds the `develop` version should use a `-dev` prefix.
So when we are working on `release/1.13.x` then `develop` is at `1.14.0-dev`.

If you wanted to be really fancy the `release/1.13.x` branch should start out as `1.13.0-rc` or `1.13.0-rc.0` and then switch it to `1.13.0` proper once we consider it stable, and then release. But that kinda affects the test flow, where the final build is different from the one we tested and consider stable. But then again the only change would be the version, so it should be sable in the same way.

Reply:
I wrote the scripts and updated the docs, would you mind having a look?